pkg/crc/preflight/preflight_checks_tray_windows.go

```diff
@@ -41,26 +40,9 @@ func fixDaemonServiceInstalled() error {
 	if err != nil {
 		return fmt.Errorf("Unable to find the current executables location: %v", err)
 	}
-	binPathWithArgs := fmt.Sprintf("%s daemon --log-level debug", strings.TrimSpace(binPath))
+	binPathWithArgs := fmt.Sprintf("%s daemon %s", strings.TrimSpace(binPath), constants.GetHomeDir())
 
-	// get the account name
-	whoami, err := exec.LookPath("whoami.exe")
-	if err != nil {
-		return fmt.Errorf("Unable to find whoami.exe in path: %v", err)
-	}
-	stdOut, err := exec.Command(whoami).Output() // #nosec
-	if err != nil {
-		return fmt.Errorf("Unable to get the current account name: %v", err)
-	}
-	accountName := string(stdOut)
-	accountName = strings.TrimSpace(accountName)
-
-	// get the password from user
-	password, err := input.PromptUserForSecret("Enter account login password for service installation", "This is the login password of your current account, needed to install the daemon service")
-	if err != nil {
-		return fmt.Errorf("Unable to get login password: %v", err)
-	}
-	err = service.Create(constants.DaemonServiceName, binPathWithArgs, accountName, password)
+	err = service.Create(constants.DaemonServiceName, binPathWithArgs)
```

Comment:
Is the daemon service going to run with elevated privileges compared to the user running `crc setup`?

Reply:
Yes, the daemon service will run with the privileges of the `LocalSystem`account which is there just for service and has extensive privileges on the system..

Reply:
It's a bit worrying to run something with elevated privileges,  and to be able to configure where it's going to write on the system (through the "home dir" argument), even if I don't really have a specific situation where this could be abused.

Reply:
What could an alternative to this be?

Reply:
Yes, to change the services config "home dir" argument it needs admin privileges.

Reply:
I really don't know what alternatives there are, I haven't looked closely at that code and how the daemon on Windows works. However your question made me realize that the commit log is not explaining _why_ we want to run the daemon under a LocalSystem account (ie c&p most of #1286, and maybe mention that currently when trying to start the service, some users get the error described in https://www.opcsupport.com/s/article/How-do-I-configure-a-user-account-to-have-logon-as-a-service-permissions )